api/integrations/twitter/twitter-bot

```diff
@@ -28,12 +28,14 @@ import os
 import sys
 import optparse
 import six.moves.configparser
+from six.moves.configparser import ConfigParser
```

Comment:
We may want a follow-up commit that eliminates the prior `import six.moves.configparser` statement.